src/rebar_file_utils.erl

```diff
@@ -198,8 +216,10 @@ delete_each_dir_win32([Dir | Rest]) ->
     delete_each_dir_win32(Rest).
 
 xcopy_win32(Source,Dest)->
+    %% "xcopy \"~s\" \"~s\" /q /y /e 2> nul", Chanegd to robocopy to
+    %% handle long names. May have issues with older windows.
     {ok, R} = rebar_utils:sh(
-                ?FMT("xcopy \"~s\" \"~s\" /q /y /e 2> nul",
+                ?FMT("robocopy \"~s\" \"~s\" /e /is 2> nul",
                      [filename:nativename(Source), filename:nativename(Dest)]),
```

Comment:
Nah. Spaces are more common than `"` in paths. We have the same problem elsewhere apparently: https://github.com/Taure/rebar3/blob/dw-windows/src/rebar_file_utils.erl#L111-L112 so that's out of scope for now I guess. I'll create an issue for it.